service/service.go

```diff
@@ -57,6 +59,14 @@ type Settings struct {
 	// LoggingOptions provides a way to change behavior of zap logging.
 	LoggingOptions []zap.Option
 
+	OtelMetricViews []sdkmetric.View
```

Comment:
I know @mar4uk has been following the discussions around instrumenting the collector itself. Perhaps she could provide some feedback to this part of the PR?